toolversions.sh

```diff
@@ -42,6 +42,7 @@ install_docfx() {
   if [[ ! -f $DOCFX ]]
   then
     (echo "Fetching docfx v${DOCFX_VERSION}";
+     mkdir -p $REPO_ROOT/packages;
```

Comment:
why not use nuget like the other dependencies?

https://www.nuget.org/packages/docfx/